test/integration/custom-deployment-bucket.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const uuid = require('uuid');
+const { expect } = require('chai');
+const fixtures = require('../fixtures/programmatic');
+const awsRequest = require('@serverless/test/aws-request');
+const { deployService, removeService } = require('../utils/integration');
+const { createBucket, deleteBucket } = require('../utils/s3');
+
+describe('Base AWS provider test', function () {
+  this.timeout(1000 * 60 * 10);
+
+  const bucketName = `serverless-test-${uuid.v4()}`;
+
+  before(async () => {
+    await createBucket(bucketName);
+  });
+
+  it('should deploy in the configured aws bucket', async () => {
+    // we cannot deploy an empty fixture like aws so we go for a small one
+    const { servicePath } = await fixtures.setup('function', {
```

Comment:
Let's move `setup/teardown` of fixture to `before/after` hooks similar to other integration tests

Reply:
Should that include the `deployService/removeService`?

Reply:
Yes, sorry for not being specific here :+1: